native/cocos/application/CocosApplication.cpp

```diff
@@ -121,7 +121,7 @@ void CocosApplication::onResume() {
 }
 
 void CocosApplication::onClose() {
```

Comment:
Is it better to rename close to 'onTerminate' or 'onStop'  or 'onDestroy'?

Reply:
Sounds the same thing to me, I guess no need to rename it